### PR TITLE
Feature/ht-30 add manage billing

### PR DESCRIPTION
**Pull request by jamieyoung5**
https://github.com/jamieyoung5/haul-this/issues/50#issue-2488028219

### CODE REVIEW

**Review by conor27nash — APPROVED** [review]
looks all good to me

File: HaulThis/ViewModels/ListViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+public class ListViewModel<TItem, TService> : INotifyPropertyChanged
+{
+    protected readonly TService _service;
+
+    public ObservableCollection<TItem> Items { get; private set; } = [];
+
+    public ListViewModel(TService service)
+    {
+        _service = service;
+        LoadItems();
```
Comment (stewartbebbington) [inline HaulThis/ViewModels/ListViewModel.cs:16]: could maybe have support for cancellation tokens to make the LoadItems method more responsive? E.g user navigates away from the page there might be a delay.

If this is too much effort we can ignore this for now

File: HaulThis/ViewModels/ListViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.ObjectModel;
```
Comment (stewartbebbington) [inline HaulThis/ViewModels/ListViewModel.cs:1]: more error handling in here?

Comment (jamieyoung5) [reply HaulThis/ViewModels/ListViewModel.cs:1]: Will fix in a follow up pr

**Review by stewartbebbington — APPROVED** [review]

**Pull request merged**